INFO:root:This message will be logged to aika.log inside the Logs folder
INFO:root:This message will be logged to aika.log inside the Logs folder
INFO:root:This message will be logged to aika.log inside the Logs folder
INFO:root:This message will be logged to aika.log inside the Logs folder
INFO:root:This message will be logged to aika.log inside the Logs folder
INFO:root:This message will be logged to aika.log inside the Logs folder
WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='nominatim.openstreetmap.org', port=443): Read timed out. (read timeout=1)")': /reverse?lat=-1.0&lon=-1.0&format=json&addressdetails=1
INFO:root:This message will be logged to aika.log inside the Logs folder
INFO:root:This message will be logged to aika.log inside the Logs folder
INFO:root:This message will be logged to aika.log inside the Logs folder
2025-04-21 11:34:52 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='nominatim.openstreetmap.org', port=443): Read timed out. (read timeout=1)")': /reverse?lat=14.11436&lon=120.814495&format=json&addressdetails=1
2025-04-21 11:56:38 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='nominatim.openstreetmap.org', port=443): Read timed out. (read timeout=1)")': /reverse?lat=14.021553&lon=121.605433&format=json&addressdetails=1
2025-04-21 11:56:40 - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='nominatim.openstreetmap.org', port=443): Read timed out. (read timeout=1)")': /reverse?lat=14.021553&lon=121.605433&format=json&addressdetails=1
2025-04-21 11:56:44 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='nominatim.openstreetmap.org', port=443): Read timed out. (read timeout=1)")': /reverse?lat=14.021553&lon=121.605433&format=json&addressdetails=1
2025-04-21 12:03:40 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='nominatim.openstreetmap.org', port=443): Read timed out. (read timeout=1)")': /reverse?lat=11.323742&lon=124.34319&format=json&addressdetails=1
2025-04-21 12:04:31 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='nominatim.openstreetmap.org', port=443): Read timed out. (read timeout=1)")': /reverse?lat=10.945495&lon=123.316393&format=json&addressdetails=1
2025-04-21 12:04:33 - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='nominatim.openstreetmap.org', port=443): Read timed out. (read timeout=1)")': /reverse?lat=10.945495&lon=123.316393&format=json&addressdetails=1
2025-04-21 12:04:38 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='nominatim.openstreetmap.org', port=443): Read timed out. (read timeout=1)")': /reverse?lat=10.945495&lon=123.316393&format=json&addressdetails=1
2025-04-21 12:04:39 - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='nominatim.openstreetmap.org', port=443): Read timed out. (read timeout=1)")': /reverse?lat=10.945495&lon=123.316393&format=json&addressdetails=1
2025-04-21 12:04:45 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='nominatim.openstreetmap.org', port=443): Read timed out. (read timeout=1)")': /reverse?lat=16.061978&lon=119.870812&format=json&addressdetails=1
2025-04-21 12:04:47 - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='nominatim.openstreetmap.org', port=443): Read timed out. (read timeout=1)")': /reverse?lat=16.061978&lon=119.870812&format=json&addressdetails=1
2025-04-21 12:04:51 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='nominatim.openstreetmap.org', port=443): Read timed out. (read timeout=1)")': /reverse?lat=16.061978&lon=119.870812&format=json&addressdetails=1
2025-04-21 12:04:53 - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='nominatim.openstreetmap.org', port=443): Read timed out. (read timeout=1)")': /reverse?lat=16.061978&lon=119.870812&format=json&addressdetails=1
2025-04-21 12:04:59 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='nominatim.openstreetmap.org', port=443): Read timed out. (read timeout=1)")': /reverse?lat=10.357817&lon=123.91911&format=json&addressdetails=1
2025-04-21 12:05:01 - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='nominatim.openstreetmap.org', port=443): Read timed out. (read timeout=1)")': /reverse?lat=10.357817&lon=123.91911&format=json&addressdetails=1
2025-04-21 12:05:05 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='nominatim.openstreetmap.org', port=443): Read timed out. (read timeout=1)")': /reverse?lat=10.357817&lon=123.91911&format=json&addressdetails=1
2025-04-21 12:05:07 - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='nominatim.openstreetmap.org', port=443): Read timed out. (read timeout=1)")': /reverse?lat=10.357817&lon=123.91911&format=json&addressdetails=1
2025-04-21 12:05:13 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='nominatim.openstreetmap.org', port=443): Read timed out. (read timeout=1)")': /reverse?lat=15.652652&lon=119.931282&format=json&addressdetails=1
2025-04-21 12:05:15 - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='nominatim.openstreetmap.org', port=443): Read timed out. (read timeout=1)")': /reverse?lat=15.652652&lon=119.931282&format=json&addressdetails=1
2025-04-21 12:11:36 - WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPConnectionPool(host='localhost', port=54668): Read timed out. (read timeout=120)")': /session/173e0cfc36f662f38df9c2831372ef8e
2025-04-21 13:22:44 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='nominatim.openstreetmap.org', port=443): Read timed out. (read timeout=1)")': /reverse?lat=16.017313&lon=119.870227&format=json&addressdetails=1
2025-04-21 13:23:00 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='nominatim.openstreetmap.org', port=443): Read timed out. (read timeout=1)")': /reverse?lat=-1.0&lon=-1.0&format=json&addressdetails=1
2025-04-21 13:23:02 - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='nominatim.openstreetmap.org', port=443): Read timed out. (read timeout=1)")': /reverse?lat=-1.0&lon=-1.0&format=json&addressdetails=1
2025-04-21 13:23:07 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='nominatim.openstreetmap.org', port=443): Read timed out. (read timeout=1)")': /reverse?lat=-1.0&lon=-1.0&format=json&addressdetails=1
2025-04-21 13:23:08 - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='nominatim.openstreetmap.org', port=443): Read timed out. (read timeout=1)")': /reverse?lat=-1.0&lon=-1.0&format=json&addressdetails=1
2025-04-21 13:23:13 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='nominatim.openstreetmap.org', port=443): Read timed out. (read timeout=1)")': /reverse?lat=-1.0&lon=-1.0&format=json&addressdetails=1
2025-04-21 13:23:14 - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='nominatim.openstreetmap.org', port=443): Read timed out. (read timeout=1)")': /reverse?lat=-1.0&lon=-1.0&format=json&addressdetails=1
2025-04-21 13:23:19 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='nominatim.openstreetmap.org', port=443): Read timed out. (read timeout=1)")': /reverse?lat=10.898075&lon=123.417128&format=json&addressdetails=1
2025-04-21 13:23:20 - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='nominatim.openstreetmap.org', port=443): Read timed out. (read timeout=1)")': /reverse?lat=10.898075&lon=123.417128&format=json&addressdetails=1
2025-04-21 13:23:26 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='nominatim.openstreetmap.org', port=443): Read timed out. (read timeout=1)")': /reverse?lat=14.021448&lon=121.605112&format=json&addressdetails=1
2025-04-21 13:23:28 - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='nominatim.openstreetmap.org', port=443): Read timed out. (read timeout=1)")': /reverse?lat=14.021448&lon=121.605112&format=json&addressdetails=1
2025-04-21 13:23:33 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='nominatim.openstreetmap.org', port=443): Read timed out. (read timeout=1)")': /reverse?lat=14.021448&lon=121.605112&format=json&addressdetails=1
2025-04-21 13:23:34 - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='nominatim.openstreetmap.org', port=443): Read timed out. (read timeout=1)")': /reverse?lat=14.021448&lon=121.605112&format=json&addressdetails=1
2025-04-21 13:23:39 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='nominatim.openstreetmap.org', port=443): Read timed out. (read timeout=1)")': /reverse?lat=14.021448&lon=121.605112&format=json&addressdetails=1
2025-04-21 13:23:40 - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='nominatim.openstreetmap.org', port=443): Read timed out. (read timeout=1)")': /reverse?lat=14.021448&lon=121.605112&format=json&addressdetails=1
2025-04-21 13:25:47 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='nominatim.openstreetmap.org', port=443): Read timed out. (read timeout=1)")': /reverse?lat=9.356703&lon=123.275908&format=json&addressdetails=1
2025-04-21 13:25:48 - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='nominatim.openstreetmap.org', port=443): Read timed out. (read timeout=1)")': /reverse?lat=9.356703&lon=123.275908&format=json&addressdetails=1
2025-04-21 13:25:53 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='nominatim.openstreetmap.org', port=443): Read timed out. (read timeout=1)")': /reverse?lat=9.356703&lon=123.275908&format=json&addressdetails=1
2025-04-21 13:25:54 - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='nominatim.openstreetmap.org', port=443): Read timed out. (read timeout=1)")': /reverse?lat=9.356703&lon=123.275908&format=json&addressdetails=1
2025-04-21 13:25:59 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='nominatim.openstreetmap.org', port=443): Read timed out. (read timeout=1)")': /reverse?lat=9.356703&lon=123.275908&format=json&addressdetails=1
2025-04-21 13:26:00 - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='nominatim.openstreetmap.org', port=443): Read timed out. (read timeout=1)")': /reverse?lat=9.356703&lon=123.275908&format=json&addressdetails=1
2025-04-21 13:26:07 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='nominatim.openstreetmap.org', port=443): Read timed out. (read timeout=1)")': /reverse?lat=14.009267&lon=120.728573&format=json&addressdetails=1
2025-04-21 13:26:08 - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='nominatim.openstreetmap.org', port=443): Read timed out. (read timeout=1)")': /reverse?lat=14.009267&lon=120.728573&format=json&addressdetails=1
2025-04-21 13:26:13 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='nominatim.openstreetmap.org', port=443): Read timed out. (read timeout=1)")': /reverse?lat=14.009267&lon=120.728573&format=json&addressdetails=1
2025-04-21 13:26:14 - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='nominatim.openstreetmap.org', port=443): Read timed out. (read timeout=1)")': /reverse?lat=14.009267&lon=120.728573&format=json&addressdetails=1
2025-04-21 13:26:19 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='nominatim.openstreetmap.org', port=443): Read timed out. (read timeout=1)")': /reverse?lat=14.009267&lon=120.728573&format=json&addressdetails=1
2025-04-21 13:26:21 - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='nominatim.openstreetmap.org', port=443): Read timed out. (read timeout=1)")': /reverse?lat=14.009267&lon=120.728573&format=json&addressdetails=1
2025-04-21 13:26:27 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='nominatim.openstreetmap.org', port=443): Read timed out. (read timeout=1)")': /reverse?lat=15.812632&lon=121.108167&format=json&addressdetails=1
2025-04-21 13:26:28 - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='nominatim.openstreetmap.org', port=443): Read timed out. (read timeout=1)")': /reverse?lat=15.812632&lon=121.108167&format=json&addressdetails=1
2025-04-21 13:26:33 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='nominatim.openstreetmap.org', port=443): Read timed out. (read timeout=1)")': /reverse?lat=15.812632&lon=121.108167&format=json&addressdetails=1
2025-04-23 09:33:58 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='nominatim.openstreetmap.org', port=443): Read timed out. (read timeout=1)")': /reverse?lat=14.938745&lon=120.84205&format=json&addressdetails=1
2025-04-23 09:34:25 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000001CC86532E90>: Failed to establish a new connection: [WinError 10051] A socket operation was attempted to an unreachable network')': /reverse?lat=14.300597&lon=120.952342&format=json&addressdetails=1
2025-04-23 09:34:27 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000001CC86644CD0>: Failed to establish a new connection: [WinError 10051] A socket operation was attempted to an unreachable network')': /reverse?lat=16.016557&lon=119.869118&format=json&addressdetails=1
2025-04-23 09:34:28 - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000001CC86644190>: Failed to establish a new connection: [WinError 10051] A socket operation was attempted to an unreachable network')': /reverse?lat=16.016557&lon=119.869118&format=json&addressdetails=1
2025-04-23 09:34:39 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000001CC86644910>: Failed to establish a new connection: [WinError 10051] A socket operation was attempted to an unreachable network')': /reverse?lat=14.686648&lon=121.024493&format=json&addressdetails=1
2025-04-23 09:34:42 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='nominatim.openstreetmap.org', port=443): Read timed out. (read timeout=1)")': /reverse?lat=15.324415&lon=120.93635&format=json&addressdetails=1
2025-04-23 09:35:31 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000001CC86533ED0>: Failed to establish a new connection: [WinError 10051] A socket operation was attempted to an unreachable network')': /reverse?lat=10.35855&lon=123.91859&format=json&addressdetails=1
2025-04-23 09:38:01 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000268BD66F8C0>: Failed to establish a new connection: [WinError 10051] A socket operation was attempted to an unreachable network')': /reverse?lat=14.938688&lon=120.842118&format=json&addressdetails=1
2025-04-23 09:38:03 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000268BD6F51D0>: Failed to establish a new connection: [WinError 10051] A socket operation was attempted to an unreachable network')': /reverse?lat=14.938665&lon=120.842133&format=json&addressdetails=1
2025-04-23 09:38:08 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000268BD6F56D0>: Failed to establish a new connection: [WinError 10051] A socket operation was attempted to an unreachable network')': /reverse?lat=14.934912&lon=120.842967&format=json&addressdetails=1
2025-04-23 09:38:09 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000268BD6F5A90>: Failed to establish a new connection: [WinError 10051] A socket operation was attempted to an unreachable network')': /reverse?lat=14.93832&lon=120.842517&format=json&addressdetails=1
2025-04-23 09:38:13 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000268BD6F5E50>: Failed to establish a new connection: [WinError 10051] A socket operation was attempted to an unreachable network')': /reverse?lat=14.939713&lon=120.842333&format=json&addressdetails=1
2025-04-23 09:39:08 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000268BD5E3D90>: Failed to establish a new connection: [WinError 10051] A socket operation was attempted to an unreachable network')': /reverse?lat=14.820247&lon=121.04901&format=json&addressdetails=1
2025-04-23 09:39:09 - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000268BD5E2D50>: Failed to establish a new connection: [WinError 10051] A socket operation was attempted to an unreachable network')': /reverse?lat=14.820247&lon=121.04901&format=json&addressdetails=1
2025-04-23 09:39:13 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000268BD6F60D0>: Failed to establish a new connection: [WinError 10051] A socket operation was attempted to an unreachable network')': /reverse?lat=14.820247&lon=121.04901&format=json&addressdetails=1
2025-04-23 09:39:14 - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000268BD6F5E50>: Failed to establish a new connection: [WinError 10051] A socket operation was attempted to an unreachable network')': /reverse?lat=14.820247&lon=121.04901&format=json&addressdetails=1
2025-04-23 09:39:29 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000268BD6F56D0>: Failed to establish a new connection: [WinError 10051] A socket operation was attempted to an unreachable network')': /reverse?lat=9.798233&lon=123.357422&format=json&addressdetails=1
2025-04-23 09:39:36 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000268BD6F4190>: Failed to establish a new connection: [WinError 10051] A socket operation was attempted to an unreachable network')': /reverse?lat=13.774172&lon=121.115332&format=json&addressdetails=1
2025-04-23 09:40:01 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='nominatim.openstreetmap.org', port=443): Read timed out. (read timeout=1)")': /reverse?lat=15.3215&lon=120.949607&format=json&addressdetails=1
2025-04-23 09:40:02 - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000268BD6F5090>: Failed to establish a new connection: [WinError 10051] A socket operation was attempted to an unreachable network')': /reverse?lat=15.3215&lon=120.949607&format=json&addressdetails=1
2025-04-23 09:40:09 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000268BD6F4E10>: Failed to establish a new connection: [WinError 10051] A socket operation was attempted to an unreachable network')': /reverse?lat=7.910752&lon=123.811493&format=json&addressdetails=1
2025-04-23 09:40:10 - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000268BD6F51D0>: Failed to establish a new connection: [WinError 10051] A socket operation was attempted to an unreachable network')': /reverse?lat=7.910752&lon=123.811493&format=json&addressdetails=1
2025-04-23 09:40:22 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000268BD6F4050>: Failed to establish a new connection: [WinError 10051] A socket operation was attempted to an unreachable network')': /reverse?lat=9.382072&lon=123.235838&format=json&addressdetails=1
2025-04-23 09:40:24 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000268BD6F5810>: Failed to establish a new connection: [WinError 10051] A socket operation was attempted to an unreachable network')': /reverse?lat=9.333217&lon=123.294325&format=json&addressdetails=1
2025-04-23 09:40:25 - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000268BD6F5310>: Failed to establish a new connection: [WinError 10051] A socket operation was attempted to an unreachable network')': /reverse?lat=9.333217&lon=123.294325&format=json&addressdetails=1
2025-04-23 09:40:26 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000268BD5E3D90>: Failed to establish a new connection: [WinError 10051] A socket operation was attempted to an unreachable network')': /reverse?lat=11.307853&lon=124.36824&format=json&addressdetails=1
2025-04-23 09:40:27 - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000268BD5E2D50>: Failed to establish a new connection: [WinError 10051] A socket operation was attempted to an unreachable network')': /reverse?lat=11.307853&lon=124.36824&format=json&addressdetails=1
2025-04-23 09:40:33 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='nominatim.openstreetmap.org', port=443): Read timed out. (read timeout=1)")': /reverse?lat=14.114725&lon=120.814168&format=json&addressdetails=1
2025-04-23 09:40:38 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000268BD6F4910>: Failed to establish a new connection: [WinError 10051] A socket operation was attempted to an unreachable network')': /reverse?lat=15.811538&lon=119.954302&format=json&addressdetails=1
2025-04-23 09:40:46 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000268BD6F4F50>: Failed to establish a new connection: [WinError 10051] A socket operation was attempted to an unreachable network')': /reverse?lat=14.314513&lon=120.95095&format=json&addressdetails=1
2025-04-23 09:40:47 - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000268BD6F4190>: Failed to establish a new connection: [WinError 10051] A socket operation was attempted to an unreachable network')': /reverse?lat=14.314513&lon=120.95095&format=json&addressdetails=1
2025-04-23 09:40:51 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000268BD6F5A90>: Failed to establish a new connection: [WinError 10051] A socket operation was attempted to an unreachable network')': /reverse?lat=14.314513&lon=120.95095&format=json&addressdetails=1
2025-04-23 09:40:52 - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000268BD6F60D0>: Failed to establish a new connection: [WinError 10051] A socket operation was attempted to an unreachable network')': /reverse?lat=14.314513&lon=120.95095&format=json&addressdetails=1
2025-04-23 09:40:58 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000268BD6F4A50>: Failed to establish a new connection: [WinError 10051] A socket operation was attempted to an unreachable network')': /reverse?lat=14.865442&lon=120.494015&format=json&addressdetails=1
2025-04-23 09:40:59 - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000268BD6F42D0>: Failed to establish a new connection: [WinError 10051] A socket operation was attempted to an unreachable network')': /reverse?lat=14.865442&lon=120.494015&format=json&addressdetails=1
2025-04-23 09:41:03 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000268BD6F4410>: Failed to establish a new connection: [WinError 10051] A socket operation was attempted to an unreachable network')': /reverse?lat=14.865442&lon=120.494015&format=json&addressdetails=1
2025-04-23 09:41:04 - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000268BD6F6210>: Failed to establish a new connection: [WinError 10051] A socket operation was attempted to an unreachable network')': /reverse?lat=14.865442&lon=120.494015&format=json&addressdetails=1
2025-04-23 09:41:09 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000268BD6F65D0>: Failed to establish a new connection: [WinError 10051] A socket operation was attempted to an unreachable network')': /reverse?lat=14.865442&lon=120.494015&format=json&addressdetails=1
2025-04-23 09:41:10 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000268BD6F6990>: Failed to establish a new connection: [WinError 10051] A socket operation was attempted to an unreachable network')': /reverse?lat=14.025265&lon=121.596825&format=json&addressdetails=1
2025-04-23 09:41:11 - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000268BD6F6AD0>: Failed to establish a new connection: [WinError 10051] A socket operation was attempted to an unreachable network')': /reverse?lat=14.025265&lon=121.596825&format=json&addressdetails=1
2025-04-23 09:41:16 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000268BD6F6E90>: Failed to establish a new connection: [WinError 10051] A socket operation was attempted to an unreachable network')': /reverse?lat=14.025265&lon=121.596825&format=json&addressdetails=1
2025-04-23 09:41:17 - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000268BD6F6FD0>: Failed to establish a new connection: [WinError 10051] A socket operation was attempted to an unreachable network')': /reverse?lat=14.025265&lon=121.596825&format=json&addressdetails=1
2025-04-23 09:41:22 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000268BD5E3D90>: Failed to establish a new connection: [WinError 10051] A socket operation was attempted to an unreachable network')': /reverse?lat=9.377022&lon=123.236712&format=json&addressdetails=1
2025-04-23 09:41:23 - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000268BD5E2D50>: Failed to establish a new connection: [WinError 10051] A socket operation was attempted to an unreachable network')': /reverse?lat=9.377022&lon=123.236712&format=json&addressdetails=1
2025-04-23 09:41:28 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000268BD6F7250>: Failed to establish a new connection: [WinError 10051] A socket operation was attempted to an unreachable network')': /reverse?lat=9.377022&lon=123.236712&format=json&addressdetails=1
2025-04-23 09:41:29 - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000268BD6F6E90>: Failed to establish a new connection: [WinError 10051] A socket operation was attempted to an unreachable network')': /reverse?lat=9.377022&lon=123.236712&format=json&addressdetails=1
2025-04-23 09:41:30 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000268BD6F6D50>: Failed to establish a new connection: [WinError 10051] A socket operation was attempted to an unreachable network')': /reverse?lat=15.958775&lon=120.401772&format=json&addressdetails=1
2025-04-23 09:41:31 - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000268BD6F6990>: Failed to establish a new connection: [WinError 10051] A socket operation was attempted to an unreachable network')': /reverse?lat=15.958775&lon=120.401772&format=json&addressdetails=1
2025-04-23 09:41:35 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000268BD6F6850>: Failed to establish a new connection: [WinError 10051] A socket operation was attempted to an unreachable network')': /reverse?lat=15.958775&lon=120.401772&format=json&addressdetails=1
2025-04-23 09:41:37 - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000268BD6F65D0>: Failed to establish a new connection: [WinError 10051] A socket operation was attempted to an unreachable network')': /reverse?lat=15.958775&lon=120.401772&format=json&addressdetails=1
2025-04-23 09:41:41 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000268BD6F6350>: Failed to establish a new connection: [WinError 10051] A socket operation was attempted to an unreachable network')': /reverse?lat=14.009267&lon=120.728573&format=json&addressdetails=1
2025-04-23 09:41:42 - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000268BD6F5950>: Failed to establish a new connection: [WinError 10051] A socket operation was attempted to an unreachable network')': /reverse?lat=14.009267&lon=120.728573&format=json&addressdetails=1
2025-04-23 09:41:46 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000268BD6F5450>: Failed to establish a new connection: [WinError 10051] A socket operation was attempted to an unreachable network')': /reverse?lat=14.009267&lon=120.728573&format=json&addressdetails=1
2025-04-23 09:41:47 - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000268BD6F5D10>: Failed to establish a new connection: [WinError 10051] A socket operation was attempted to an unreachable network')': /reverse?lat=14.009267&lon=120.728573&format=json&addressdetails=1
2025-04-23 09:41:52 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000268BD6F5E50>: Failed to establish a new connection: [WinError 10051] A socket operation was attempted to an unreachable network')': /reverse?lat=14.009267&lon=120.728573&format=json&addressdetails=1
2025-04-23 09:41:53 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000268BD6F4190>: Failed to establish a new connection: [WinError 10051] A socket operation was attempted to an unreachable network')': /reverse?lat=14.331118&lon=121.07495&format=json&addressdetails=1
2025-04-23 09:41:54 - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000268BD6F4CD0>: Failed to establish a new connection: [WinError 10051] A socket operation was attempted to an unreachable network')': /reverse?lat=14.331118&lon=121.07495&format=json&addressdetails=1
2025-04-23 09:41:58 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000268BD6F51D0>: Failed to establish a new connection: [WinError 10051] A socket operation was attempted to an unreachable network')': /reverse?lat=14.331118&lon=121.07495&format=json&addressdetails=1
2025-04-23 09:42:00 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000268BD6F7390>: Failed to establish a new connection: [WinError 10051] A socket operation was attempted to an unreachable network')': /reverse?lat=15.812572&lon=121.10815&format=json&addressdetails=1
2025-04-23 09:42:01 - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000268BD6F6710>: Failed to establish a new connection: [WinError 10051] A socket operation was attempted to an unreachable network')': /reverse?lat=15.812572&lon=121.10815&format=json&addressdetails=1
2025-04-23 09:42:05 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000268BD6F4E10>: Failed to establish a new connection: [WinError 10051] A socket operation was attempted to an unreachable network')': /reverse?lat=15.812572&lon=121.10815&format=json&addressdetails=1
2025-04-23 09:42:11 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000268BD6F7750>: Failed to establish a new connection: [WinError 10051] A socket operation was attempted to an unreachable network')': /reverse?lat=14.683178&lon=121.021063&format=json&addressdetails=1
2025-04-23 09:42:13 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000268BD5E3D90>: Failed to establish a new connection: [WinError 10051] A socket operation was attempted to an unreachable network')': /reverse?lat=14.628657&lon=120.999413&format=json&addressdetails=1
2025-04-23 09:42:20 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000268BD6F7B10>: Failed to establish a new connection: [WinError 10051] A socket operation was attempted to an unreachable network')': /reverse?lat=16.472827&lon=121.112328&format=json&addressdetails=1
2025-04-23 09:42:22 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000268BD6F7610>: Failed to establish a new connection: [WinError 10051] A socket operation was attempted to an unreachable network')': /reverse?lat=14.683435&lon=121.020683&format=json&addressdetails=1
2025-04-23 09:42:23 - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000268BD6F4E10>: Failed to establish a new connection: [WinError 10051] A socket operation was attempted to an unreachable network')': /reverse?lat=14.683435&lon=121.020683&format=json&addressdetails=1
2025-04-23 09:42:25 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000268BD6F6710>: Failed to establish a new connection: [WinError 10051] A socket operation was attempted to an unreachable network')': /reverse?lat=13.770792&lon=121.11633&format=json&addressdetails=1
2025-04-23 09:42:27 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000268BD6F51D0>: Failed to establish a new connection: [WinError 10051] A socket operation was attempted to an unreachable network')': /reverse?lat=13.7741&lon=121.1152&format=json&addressdetails=1
2025-04-23 09:42:45 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000268BD6F4CD0>: Failed to establish a new connection: [WinError 10051] A socket operation was attempted to an unreachable network')': /reverse?lat=14.394653&lon=121.038892&format=json&addressdetails=1
2025-04-23 09:42:46 - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000268BD6F5090>: Failed to establish a new connection: [WinError 10051] A socket operation was attempted to an unreachable network')': /reverse?lat=14.394653&lon=121.038892&format=json&addressdetails=1
2025-04-23 09:45:34 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='nominatim.openstreetmap.org', port=443): Read timed out. (read timeout=1)")': /reverse?lat=9.798233&lon=123.357422&format=json&addressdetails=1
2025-04-23 09:46:04 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000268BD5E3ED0>: Failed to establish a new connection: [WinError 10051] A socket operation was attempted to an unreachable network')': /reverse?lat=10.162262&lon=124.778303&format=json&addressdetails=1
2025-04-23 09:49:35 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='nominatim.openstreetmap.org', port=443): Read timed out. (read timeout=1)")': /reverse?lat=10.038597&lon=123.438883&format=json&addressdetails=1
2025-04-23 09:50:47 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='nominatim.openstreetmap.org', port=443): Read timed out. (read timeout=1)")': /reverse?lat=14.938773&lon=120.842242&format=json&addressdetails=1
2025-04-23 09:53:51 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='nominatim.openstreetmap.org', port=443): Read timed out. (read timeout=1)")': /reverse?lat=14.682882&lon=121.020697&format=json&addressdetails=1
2025-04-23 10:06:00 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='nominatim.openstreetmap.org', port=443): Read timed out. (read timeout=1)")': /reverse?lat=14.68348&lon=121.020622&format=json&addressdetails=1
2025-04-23 10:09:06 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='nominatim.openstreetmap.org', port=443): Read timed out. (read timeout=1)")': /reverse?lat=15.812527&lon=121.108507&format=json&addressdetails=1
2025-04-23 10:09:31 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='nominatim.openstreetmap.org', port=443): Read timed out. (read timeout=1)")': /reverse?lat=10.934765&lon=123.28409&format=json&addressdetails=1
2025-04-23 10:11:12 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='nominatim.openstreetmap.org', port=443): Read timed out. (read timeout=1)")': /reverse?lat=13.770792&lon=121.11633&format=json&addressdetails=1
2025-04-23 10:15:46 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='nominatim.openstreetmap.org', port=443): Read timed out. (read timeout=1)")': /reverse?lat=15.750043&lon=119.942905&format=json&addressdetails=1
2025-04-23 10:15:50 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='nominatim.openstreetmap.org', port=443): Read timed out. (read timeout=1)")': /reverse?lat=10.898118&lon=123.417142&format=json&addressdetails=1
2025-04-23 10:18:20 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='nominatim.openstreetmap.org', port=443): Read timed out. (read timeout=1)")': /reverse?lat=9.862098&lon=123.577127&format=json&addressdetails=1
2025-04-23 10:19:47 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='nominatim.openstreetmap.org', port=443): Read timed out. (read timeout=1)")': /reverse?lat=14.445905&lon=121.052142&format=json&addressdetails=1
